solidity/dashboard/src/components/Routing.jsx

```diff
@@ -28,7 +30,6 @@ class Routing extends React.Component {
         <Route exact path="/glossary">
           <Redirect to="/resources/quick-terminology" />
         </Route>
-        <Route exact path="/grant/:grantId" component={TokenGrantPreviewPage} />
```

Comment:
Just to check---this isn't a URL that used to work before the redesign and now won't, right? If it is, let's introduce a redirect.

Reply:
I've set `TokenGrantPreviewPage` path to `/grant/:grantId` so redirect is no longer neede here.

Reply:
Is that a URL that's hooked up anywhere? Also, if we have `/token-grants` to display the list of grants, it seems reasonable to have `/token-grants/:grantId` be the place where you find a particular grant, right?

Not looking to bikeshed this further right now, we can handle this in follow-up work---but let's do a pass where we consider each URL and its place in the conceptual information hierarchy.